normandy/control/static/control/js/components/RecipeForm.js

```diff
@@ -153,10 +154,37 @@ export default composeRecipeContainer(reduxForm({
     ? props.location.state.selectedRevision
     : null;
 
+  const formatErrors = payload => {
+    let errors = payload;
+
+    if (_.isObject(payload)) {
+      const invalidFields = Object.keys(payload);
+      if (invalidFields.length > 0) {
+        errors = isNaN(invalidFields[0]) ? {} : [];
+
+        invalidFields.forEach(fieldName => {
+          errors[fieldName] = formatErrors(payload[fieldName]);
+        });
+      }
+    }
+
+    return errors;
+  };
+
+  const onSubmitFail = errors => {
+    const { dispatch } = props;
+    const actionFormErrors = errors['arguments']; // eslint-disable-line dot-notation
```

Comment:
There was previously a problem using `arguments` with redux-form because it was a reserved keyword. Now that we're processing that data and sending it as `actionFormErrors` that's not an issue any more. I've changed this to `errors.arguments`